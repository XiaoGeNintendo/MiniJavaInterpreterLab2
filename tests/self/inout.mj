void f(){
    println("Out");
}

void f(int a){
    println("Out:"+a);
}

class A{
    A(){
        f();
        f(1);
    }

    void f(){
        println("In");
    }
}

int main(){
    new A();
    return 0;
}