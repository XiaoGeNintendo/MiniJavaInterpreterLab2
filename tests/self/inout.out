In
Out:1
Process exits with the code 0.
